resources/config/default.php

```diff
@@ -23,7 +23,7 @@
     }
 }
 
-$loader->import('dist/parameters.yml');
+$loader->import('dist/parameters_sf5.yml');
```

Comment:
can we just fixed go to the sf5 version? or do we need an if/else here?

Reply:
Yes indeed, it seems I forgot to add the if/else. Should be fixed now!